### PR TITLE
support "A_LineFile" in "#Include"

### PR DESCRIPTION
**Pull request by zero-plusplus**
**By Google Translate from Japanese to English.**

Supports `A_LineFile`.

In `AutoHotkey v2`, the relative path is`% A_LineFile% \ ..` instead of `A_WorkingDir \`.
This will show that there are many other users besides me.

I've confirmed the relative path specifications [here](https://www.autohotkey.com/v2/v2-changes.htm).
> #Include is relative to the directory containing the current file by default.

### CODE REVIEW

**Comment by CoffeeChaton** [comment]
` const includeMatch = text.match(/(?<=#include).+?\.(ahk|ext)\b/i);`
`const includeMatch = text.trim().match(/(?<=#include).+?\.(ahk|ext)\b/i);`

```
{
    #Include fn_img_main.ahk
    #Include fn_img_main.ahk
    #Include fn_time_box.ahk
    #Include fn_task.ahk
    #Include fn_music_evaluation.ahk
    ;...something
    ;......something
}
```
This code is the working at ahk v1.

**Comment by zero-plusplus** [comment]
**By Google Translate from Japanese to English.**

The way I communicated was bad. I can't read your reply intent.

I will briefly explain.
With this change, you will be able to open the following paths.
`#Include %A_LineFile%\..\otherscript.ahk`

No further changes.

The reason I talked about relative paths is because I wanted to tell you the rationale for merging this change.
This change is important to me.

I wanted to convey the following:
It is standardized in `v2` because many people use `%A_LineFile%\..` as a path prefix.
In other words, many people are expecting this change.

**Comment by cweijan** [comment]
Hi, will not using parent? `document.uri.path` cannot open the file correctly, variable parent actually is current script folder path.
image

**Comment by cweijan** [comment]
Oh, I got your mean.
In addition, A_ScriptDir also exists in AhkV2.
